limits_tools/prepare_datacard.py

```diff
@@ -0,0 +1,226 @@
+import subprocess
+from sys import exit
+from optparse import OptionParser
+import os
+
+from HiggsAnalysis.CombinedLimit.DatacardParser import *
```

Comment:
Will I need to install something to use these imports?

Reply:
Yes, I should write instructions on how to calculate limits... I'll do that tomorrow or next week.